src/components/ShortcutModal/index.tsx

```diff
@@ -7,15 +7,27 @@ import { size, zIndex } from "constants/tokens";
 import { useKeyboardShortcut, useOnClickOutside } from "hooks";
 
 const shortcuts = [
-  { keys: ["CTRL", "F"], description: "Focus on the search input" },
   {
-    keys: ["CTRL", "S"],
+    keys: [
+      ["CTRL", "F"],
+      ["CMD", "F"],
+    ],
+    description: "Focus on the search input",
+  },
+  {
+    keys: [
+      ["CTRL", "S"],
+      ["CMD", "S"],
+    ],
     description: "Toggle between search, filter, and highlight",
   },
-  { keys: ["]"], description: "Toggle the filter panel" },
-  { keys: ["N"], description: "Paginate forward to the next search result" },
+  { keys: [["]"]], description: "Toggle the filter panel" },
```

Comment:
Oops my bad I should have noticed this; let's rename this like in the wiki! I think we went with `side panel`?